### PR TITLE
Adjust fontsize up on highdpi displays

### LINKED ISSUE #3597: Incorrect font size for cards
<b>System Information:</b>
  

  
__________________________________________________________________________________________

  

When using the latest version, 2.7.0, and it is run in macOS with a Retina display, the text is much too small to read legibly. In fact, it’s half as high as before!

A work around is doubling the text size, but the bug is that the text system seems to not understand what retina graphics means. 12pt in a window in low res mode should physically be the same size as 12pt in a window in retina mode.

Can you make it so the fonts that overlay onto card images are correctly sized at 12pt?

### PR DESCRIPTION
**Pull request by basicer**
## Short roundup of the initial problem
Fix #3597

## What will change with this Pull Request?
- Scale the font correctly.

### CODE REVIEW

**Comment by ebbit1q** [comment]
duplicate of https://github.com/Cockatrice/Cockatrice/issues/3597

**Comment by ZeldaZach** [comment]
This is a pull request sir lol

**Comment by ebbit1q** [comment]
I NEED SLEEP

File: cockatrice/src/abstractgraphicsitem.cpp [hunk 1]
```diff
@@ -49,8 +49,11 @@ void AbstractGraphicsItem::paintNumberEllipse(int number,
 
     painter->restore();
 }
-void resetPainterTransform(QPainter *painter)
+#include <qDebug>
+int resetPainterTransform(QPainter *painter)
 {
     painter->resetTransform();
-    painter->setTransform(painter->deviceTransform().inverted());
-}
```
Comment (ebbit1q) [inline cockatrice/src/abstractgraphicsitem.cpp]: I see you like line endings.

**Pull request merged**